### ISSUE TITLE
java.net.URISyntaxException: Illegal character in path

### ISSUE THREAD

**Issue by paulpach** [ktoso/maven-git-commit-id-plugin#305]
Using version 2.2.2,   in windows when I build my app I get this error:

```
[INFO] [ERROR] Failed to execute goal pl.project13.maven:git-commit-id-plugin:2.2.2:revision (default) on project TwitterStreamer: Could not complete Mojo execution... java.net.URISyntaxException: Illegal character in path at index 10: file:///C:\Users\557659\SnapAll -> [Help 1]
```
The same exact build works fine in my mac.

the relevant part of the pom is this:

```xml
			<plugin>
				<groupId>pl.project13.maven</groupId>
				<artifactId>git-commit-id-plugin</artifactId>
				<configuration>
					<verbose>false</verbose>
				</configuration>
			</plugin>
```

It looks similar to #274 I don't know if this is a dup because the error message is slightly different.

**Comment by TheSnoozer** [comment]
Thanks for reporting I'll look into this asap.

**Comment by TheSnoozer** [comment]
This will indeed break the plugin execution!
I have added ```^file:///(.*)``` as exclusion pattern for the checks if it contains any passwords...
As a side note a linux path (e.g. file:///path/to/repo.git/) could been parsed without any issue....

TLDR; thanks for reporting this issue! it was not a duplicate and will be fixed with 2.2.3.
